### PR TITLE
Add release-metadata updating script

### PR DESCRIPTION
**Pull request by rell-fastly**

### CODE REVIEW

**Comment by rell-fastly** [comment]
Couldn't add releng directly for some reason to the reviewers.

File: scripts/update-release-metadata.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  client = boto3.client('s3')
+  resp = client.put_object(
+      Body=data,
+      Bucket='release-metadata',
+      Key='release-versions'
+  )
+
+  if resp.ResponseMetadata.HTTPStatusCode != 200:
+    sys.stderr.write('Unable to upload file.  Dumping response metadata.\n')
+    print(resp, file=sys.stderr)
```
Comment (leaf-fastly) [inline scripts/update-release-metadata.py:38]: Just a sanity check, does this metadata risk having any auth info in it? If so, we shouldn't be dumping it in the output of actions (especially in a public repo).

**Review by leaf-fastly — APPROVED** [review]
Just one comment about dumping from non-200 responses.

File: scripts/update-release-metadata.py (hunk 1 above)
Comment (rell-fastly) [reply scripts/update-release-metadata.py:38]: I don't think so, the only identifiable information is a hash of the uploaded object.

Comment (rell-fastly) [reply scripts/update-release-metadata.py:38]: For reference,
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Client.put_object

**Pull request merged**